app/controllers/users.js

```diff
@@ -1,13 +1,22 @@
 const { signUp, signIn } = require('../mappers/users');
+const { dataBasic, listUsers } = require('../serializers/users');
 
-const { saveUser, verifyCredentials, getAllUsers } = require('../services/users');
+const { saveUser, verifyCredentials, getAllUsers, saveUserAdmin } = require('../services/users');
 const { createToken } = require('../helpers/jwt');
 
 exports.createUser = async (req, res) => {
   try {
-    const dataUser = signUp(req.body);
-    const response = await saveUser(dataUser);
-    res.status(200).send(response);
+    const response = await saveUser(signUp(req.body));
+    res.status(200).send(dataBasic(response));
+  } catch (err) {
+    res.status(400).send(err);
+  }
+};
+
+exports.createUserAdmin = async (req, res) => {
+  try {
+    const dataUser = await saveUserAdmin(signUp(req.body));
```

Comment:
I think that you could use methods that you already have in your service for this new functionality without having to concentrate all the logic within a new method in the service.